### PR TITLE
Setup and editing instructions for technical reports

### PR DESCRIPTION
**Pull request by c1rrus**
This PR:

* Adds a `README` in the technical reports directory with instructions on how to preview and edit them.
* Adds the "Auto Markdown TOC" VSCode extension to this repo's recommended extensions as I used it to generate the table of contents in the `README`.
* Adds a link to our deployed technical reports in the main `README`

### CODE REVIEW

File: technical-reports/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- [Node.js](https://nodejs.org/en/) >= 16.8
+  - If you use [`nvm`](https://github.com/nvm-sh/nvm), you can run `nvm install` to install and use the recommended version of Node.
+- [Yarn](https://yarnpkg.com/)
+
+## Initial setup
+
+After cloning this repo, you need to install the dependencies:
+
+```
+yarn
```
Comment (kaelig) [inline technical-reports/README.md:28]: I switched the repo to npm yesterday, I should have given you a heads up, sorry!

File: technical-reports/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+```html
+<section
+  data-include="./file-format.md"
+  data-include-format="markdown"
+></section>
+```
+
+## Deployments
+
+Any changes to the source files in this directory that get merged into `main` are automatically deployed to [`https://design-tokens.github.io/community-group/`](https://design-tokens.github.io/community-group/) via the [`technical-reports` Github Action](../.github/workflows/technical-reports.yml). They are hosted using Github Pages and the build output can be found in the [`gh-pages` branch](https://github.com/design-tokens/community-group/tree/gh-pages).
```
Comment (kaelig) [inline technical-reports/README.md:56]:
```suggestion
Any changes to the source files in this directory that get merged into `main` are automatically deployed to [`https://design-tokens.github.io/community-group/`](https://design-tokens.github.io/community-group/) via the [`technical-reports` Github Action](../.github/workflows/technical-reports.yml). They are hosted using GitHub Pages and the build output can be found in the [`gh-pages` branch](https://github.com/design-tokens/community-group/tree/gh-pages).
```

💅

**Review by kaelig — APPROVED** [review]
Thank you, this should be enough information to get started. I like how you mentioned ReSpec and explained the reasoning behind the file structure.

A couple of edits and you can merge!

File: technical-reports/README.md (hunk 1 above)
Comment (c1rrus) [reply technical-reports/README.md:28]: No worries. (I was meaning to ask if we really need `yarn` here anyway, so glad to hear you've made that change)

File: technical-reports/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# DTCG Technical Reports
```
Comment (kaelig) [inline technical-reports/README.md:1]: How would someone stumble upon this? Should we mention this directory in the root README and the contributing doc?

**Comment by c1rrus** [comment]
@kaelig I've done the requested edits and added a link from the main `README`. Let me know what you think.

File: README.md [hunk 4]
```diff
@@ -18,6 +18,8 @@ We believe that a common way to share design tokens will unlock efficiency oppor
 
 [Read the charter in full](https://github.com/design-tokens/community-group/blob/main/CHARTER.md).
 
+[View the latest technical reports](https://design-tokens.github.io/community-group/) ([Instructions for editing technical reports](./technical-reports/README.md)).
```
Comment (kaelig) [inline README.md:21]:
```suggestion
[Browse the latest technical reports](https://design-tokens.github.io/community-group/) ([Instructions for editing technical reports](./technical-reports/README.md)).
```

**Review by kaelig — APPROVED** [review]
Good stuff! One small inclusive writing edit and this is good to go!

**Pull request merged**